scripts/stop-proxy.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+set -ex
```

Comment:
```suggestion
set -x
```

then you can drop the `|| true`